### PR TITLE
Add cluster autoscaler support for CLI

### PR DESCRIPTION
**Pull request by marwanad**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
@qwordy please help with this PR.

**Comment by xizhamsft** [comment]
@qwordy @yonzhan , we need this PR for S160 milestone (for Ignite). Could you make sure to merge it for the Ignite release.

**Comment by yonzhan** [comment]
@qwordy  please help merge it before Ignite release.

File: src/azure-cli/azure/cli/command_modules/acs/custom.py [hunk 1]
```diff
@@ -2920,6 +2968,21 @@ def _check_cluster_autoscaler_flag(enable_cluster_autoscaler,
...
+    """
+    if min_count is None or max_count is None:
+        if is_enable_or_update:
+            raise CLIError('Please specify both min-count and max-count when --enable-cluster-autoscaler or '
+                           '--update-cluster-autoscaler is set.')
+    if min_count is not None and max_count is not None:
+        if int(min_count) > int(max_count):
+            raise CLIError('value of min-count should be less than or equal to value of max-count.')
+        if int(node_count) < int(min_count) or int(node_count) > int(max_count):
+            raise CLIError("current node count '{}' is not in the range of min-count and max-count.".format(node_count))
```
Comment (qwordy) [inline src/azure-cli/azure/cli/command_modules/acs/custom.py]: Minor problem: Sometimes you capitalize the first letter, sometimes you use lowercase.

Comment (marwanad) [reply src/azure-cli/azure/cli/command_modules/acs/custom.py]: I made it consistent. I left it lower case when the sentence starts with the flag name.

**Comment by marwanad** [comment]
@qwordy  should be good to go now

**Comment by qwordy** [comment]
> 
> 
> @qwordy should be good to go now

Thanks for making the changes.

**Pull request merged**